packages/mobile/src/components/ContactCircle.tsx

```diff
@@ -52,9 +38,8 @@ function ContactCircle({ contact, size, thumbnailPath, name, address, style }: P
     }
 
     const fontColor = getAddressForegroundColor(address || '0x0')
-    // Mobile # is what default display name when contact isn't saved
-    if (displayName && displayName !== 'Mobile #') {
```

Comment:
Yeah! This is just checking that we don't have an actual name for the recipient, because the name field used to be mandatory and we would save a display name in the name variable. Now we can directly check if the name variable exists or not